test/integration/framework.go

```diff
@@ -273,13 +275,43 @@ func initForwardingPipelineConfig() {
 	}
 }
 
+// initCounterValues is a helper function that initializes counters values to 1.
+// The initialization is required to check if counter cells are properly cleared
+// on session establishment by PFCP Agent.
+func mustInitCountersWithDummyValue() {
+	p4rtClient, err := providers.ConnectP4rt("127.0.0.1:50001", true)
+	if err != nil {
+		panic("Cannot init counter values: " + err.Error())
+	}
+	defer providers.DisconnectP4rt()
+
+	igCounterName := p4constants.GetCounterIDToNameMap()[p4constants.CounterPreQosPipePreQosCounter]
+	egCounterName := p4constants.GetCounterIDToNameMap()[p4constants.CounterPostQosPipePostQosCounter]
+
+	for i, maxSize := uint64(0), p4constants.CounterSizePreQosPipePreQosCounter; i < maxSize; i++ {
+		dummyValue := &v1.CounterData{
+			ByteCount:   1,
+			PacketCount: 1,
+		}
+
+		if err := p4rtClient.ModifyCounterEntry(igCounterName, int64(i), dummyValue); err != nil {
+			panic(err)
+		}
+
+		if err := p4rtClient.ModifyCounterEntry(egCounterName, int64(i), dummyValue); err != nil {
+			panic(err)
+		}
+	}
```

Comment:
I think there is a way to do a wildcard write, but since the number of counters is small (1024) and this is in a test, it should be fine.